.meta.toml

```
@@ -0,0 +1,15 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/buildout-recipe
+[meta]
+template = "buildout-recipe"
+commit-id = "519ffffe1b360ede58b32a375b5f8b26059d8016"
+
+[python]
+with-appveyor = false
+with-pypy = true
+with-legacy-python = true
+with-docs = false
+with-sphinx-doctests = false
+
+[coverage]
+fail-under = 95
```

Coveralls shows currently > 96%. Does it show less when run locally?
```suggestion
fail-under = 96
```

---

Yes, https://github.com/zopefoundation/zc.recipe.cmmi/pull/5/checks?check_run_id=1936735755 shows `95.92`